storage/PreloaderThread.cpp

```diff
@@ -49,37 +49,46 @@ void PreloaderThread::run() {
   std::size_t blocks_loaded = 0;
 
   for (const CatalogRelation &relation : database_) {
-    if (relation.hasPartitionScheme()) {
+    if (relation.hasPartitionScheme() && relation.hasNUMAPlacementScheme()) {
```

Comment:
It seems that when `QUICKSTEP_HAVE_LIBNUMA` is off, `CatalogRelation` will have no definition of `hasNUMAPlacementScheme()` and will incur compile error.